### PR TITLE
More feedback

### PR DESCRIPTION
**Pull request by gcase555**
# What does this change
Addresses additional PR cleanup feedback that the upstream requested. Mostly some small touch ups to the docs, getting more linting checks to pass and adding some simple functions for repo slug extraction.

I also updated in new changes from the upstream to resolve the merge conflict in go.sum

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+version: 2
```
Comment (gcase555) [inline .goreleaser.yml:1]: these are changes from the upstream

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,50 @@
 # Changelog
```
Comment (gcase555) [inline CHANGELOG.md:1]: changes from the upstream

File: cmd/platform.go [hunk 3]
```diff
@@ -37,7 +37,7 @@ func configurePlatform(cmd *cobra.Command) {
 	flags.BoolP("ssh-auth", "", false, `Use SSH cloning URL instead of HTTPS + token. This requires that a setup with ssh keys that have access to all repos and that the server is already in known_hosts.`)
```
Comment (gcase555) [inline cmd/platform.go:37]: our changes

File: docs/README.template.md [hunk 4]
```diff
@@ -133,7 +133,12 @@ All configuration in multi-gitter can be done through command line flags, config
```
Comment (gcase555) [inline docs/README.template.md:133]: our changes

File: examples/go/upgrade-go-version.sh [hunk 5]
```diff
@@ -2,5 +2,5 @@
```
Comment (gcase555) [inline examples/go/upgrade-go-version.sh:2]: our changes

File: go.mod [hunk 6]
```diff
@@ -12,21 +12,21 @@ require (
 	github.com/mitchellh/mapstructure v1.5.0
 	github.com/pkg/errors v0.9.1
```
Comment (gcase555) [inline go.mod:13]: changes from upstream

File: internal/multigitter/run.go [hunk 7]
```diff
@@ -10,6 +10,7 @@ import (
 	"regexp"
```
Comment (gcase555) [inline internal/multigitter/run.go:10]: changes from upstream

File: internal/scm/bitbucketcloud/bitbucket_cloud.go [hunk 8]
```diff
@@ -29,7 +29,8 @@ type BitbucketCloud struct {
 	bbClient     *bitbucket.Client
```
Comment (gcase555) [inline internal/scm/bitbucketcloud/bitbucket_cloud.go:29]: our changes

File: internal/scm/bitbucketcloud/custom_structs.go [hunk 9]
```diff
@@ -56,13 +56,13 @@ type commit struct {
```
Comment (gcase555) [inline internal/scm/bitbucketcloud/custom_structs.go:56]: our changes

**Review by ngandhi64 — APPROVED** [review]

**Review by byott — APPROVED** [review]

**Pull request merged**